components/select/src/single-select/input.js

```diff
@@ -24,7 +24,7 @@ const Input = ({
     inputMaxHeight,
 }) => {
     const hasSelection = selected && typeof selected === 'string'
-    const onClear = (_, e) => {
+    const onClear = (e) => {
```

Comment:
This doesn't appear to be correct, see here: https://github.com/dhis2/ui/blob/4643200c1adfdd40223e0b4d04b144b7d9a25a4a/components/button/src/button/button.js#L39

* Callback is passed to `InputClearButton`
* `InputClearButton` passes the callback to `Button`
* `Button` calls the onClick callback with the event as second argument

Reply:
@Mohammer5: won't this be okay though in the sense that now @cooper-joe is using the native <button> element rather than our <Button> component in <InputClearButton> ? We don't seem to allow users to pass their own `onClear` down, so this seems like it would be fine to me?

Reply:
I can't follow. What I can see, is:

* The select's input component defines the `onClear` function, which is called when the user clicks the clear button.
* The `onClear` function will call the `onChange` callback provided by the user
* With the currently proposed implementation, the second parameter wouldn't be an event

Reply:
I was under the impression that this was OK, as @tomzemp says the button is now a `<buttton>` rather than a `<Button>`.

Reply:
Ah yes, my bad! When I used GH to go to the definition, it sent me to the master branch... Sorry!